learn/src/main/java/org/navgurukul/learn/courses/db/CoursesDatabase.kt

```diff
@@ -141,6 +141,32 @@ val MIGRATION_3_4 = object: Migration(3, 4) {
 
 }
 
+val MIGRATION_4_5 = object: Migration(4, 5) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+
+        //drop old table
+        database.execSQL("DROP TABLE course_exercise")
+
+        //create new table
+        database.execSQL("CREATE TABLE `course_exercise`(" +
+                " `content` TEXT," +
+                " `courseId` TEXT," +
+                " `githubLink` TEXT, " +
+                " `id` TEXT NOT NULL," +
+                " `name` TEXT," +
+                " `parentExerciseId` TEXT," +
+                " `reviewType` TEXT," +
+                " `sequenceNum` TEXT," +
+                " `slug` TEXT," +
+                " `solution` TEXT," +
```

Comment:
Lets confirm if these fields are needed by the backend team , would be a good time to clean this response